ui/src/app/memberships/page.tsx

```diff
@@ -1,6 +1,17 @@
 import { Tabs, TabsContent, TabsList, TabsTrigger } from '@/components/ui/tabs';
+import { membershipResults, optInGroupingPaths } from '@/lib/fetchers';
+import dynamic from 'next/dynamic';
+import GroupingsTableSkeleton from '@/components/table/groupings-table/groupings-table-skeleton';
+
+const MembershipsTable = dynamic(() => import('@/app/memberships/_components/memberships-table'), {
+    ssr: false,
+    loading: () => <GroupingsTableSkeleton />
+});
```

Comment:
Instead of importing with the dynamic function, you can export it with the dynamic function. You can reference the bottom of the groupings-table.tsx file.